ansible/roles/10-ui/meta/main.yaml

```diff
@@ -6,28 +6,35 @@ dependencies:
       group_name: ui
       log_file_name: de-ui.xml
       src: ui.logback.xml.j2
+      tags:
+        - logging_config
 
     - role: cfg-service
       service_name_short: "{{de.http_server.service_name_short}}"
       group_name: ui
       src: ui/nginx.conf.j2
-      dest: /etc/nginx/nginx.conf
+      dest: "{{nginx_config}}/nginx.conf"
```

Comment:
The variables used for this change need to be available at the `group_var` level, because this file is utilized in `ansible/roles/cfg-systemd-unit/templates/de-ui-nginx.service.j2`. 
```
de:
   ...
   http_server:
      ...
      config_dir: "{{de_config_dir}}/nginx"
```
Then, the line above will become `dest: "{{de.http_server.config_dir}}/nginx.conf`.

You would also have to change the volumes declaration in the nginx systemd file.
The line 
```-v /etc/nginx/nginx.conf:/etc/nginx/nginx.conf ``` could become 
```-v {{de.http_server.config_dir}}/nginx.conf:/etc/nginx/nginx.conf```